src/me.rs

```diff
@@ -264,8 +264,14 @@ fn full_search(
   lowest_cost: &mut u32, po: &PlaneOffset, step: usize, bit_depth: usize,
   lambda: u32, pmv: &[MotionVector; 2], allow_high_precision_mv: bool
 ) {
+    let search_range_y = (y_lo..=y_hi).step_by(step);
+    let search_range_x = (x_lo..=x_hi).step_by(step);
+    let search_area = search_range_y.flat_map(|y| { search_range_x.clone().map(move |x| (y, x)) });
+/*
```

Comment:
Not a fan of having commented code in the codebase. We have git history if we need to revert it. :slightly_smiling_face:

Reply:
I left it as note on what the iterator was doing, but you are right, patch amended :)